09-10-21 14:00:28: Namespace(DeepJDOT_head=False, DefRec_dist='volume_based_voxels', DefRec_on_src=False, DefRec_on_trgt=True, DefRec_weight=0.5, alpha=0.6, apply_PCM=True, batch_size=16, dataroot='/scratch/shapenets/pointda', dropout=0.5, epochs=150, exp_name='DefRec_PCM', gpus=[0], lr=0.001, mixup_params=1.0, model='dgcnn', momentum=0.9, num_regions=3, optimizer='ADAM', out_path='./experiments', seed=1, softmax=True, src_dataset='shapenet', supervised=True, test_batch_size=32, trgt_dataset='scannet', use_DeepJDOT=True, wd=5e-05)
09-10-21 14:00:28: Using GPUs [0], from 1 devices available
09-10-21 14:00:28: number of train examples in shapenet: 17378
09-10-21 14:00:28: Occurrences count of classes in shapenet train set: {0: 599, 1: 167, 2: 310, 3: 1076, 4: 4612, 5: 1620, 6: 762, 7: 158, 8: 2198, 9: 5876}
09-10-21 14:00:29: number of train examples in scannet: 6110
09-10-21 14:00:29: Occurrences count of classes in scannet train set: {0: 98, 1: 329, 2: 464, 3: 650, 4: 2578, 5: 161, 6: 210, 7: 88, 8: 495, 9: 1037}
09-10-21 14:00:29: number of test examples in scannet: 1769
09-10-21 14:00:29: Occurrences count of classes in scannet test set: {0: 26, 1: 85, 2: 146, 3: 149, 4: 801, 5: 41, 6: 61, 7: 25, 8: 134, 9: 301}
09-10-21 14:00:29: Occurrences count of classes in source shapenet train part: {0: 480, 1: 134, 2: 248, 3: 860, 4: 3690, 5: 1296, 6: 610, 7: 126, 8: 1758, 9: 4702}
09-10-21 14:00:29: Occurrences count of classes in source shapenet validation part: {0: 119, 1: 33, 2: 62, 3: 216, 4: 922, 5: 324, 6: 152, 7: 32, 8: 440, 9: 1174}
09-10-21 14:00:29: Occurrences count of classes in target scannet train part: {0: 80, 1: 270, 2: 388, 3: 517, 4: 2070, 5: 130, 6: 164, 7: 71, 8: 391, 9: 807}
09-10-21 14:00:29: Occurrences count of classes in target scannet validation part: {0: 18, 1: 59, 2: 76, 3: 133, 4: 508, 5: 31, 6: 46, 7: 17, 8: 104, 9: 230}

























































































































































305it [05:07,  1.01s/it]
09-10-21 14:05:49: Trn - Source 0, total loss: 1.0476, mixup loss: 1.0476
09-10-21 14:05:49: Trn - Target 0, DefRec loss: 2.3895
09-10-21 14:05:49: Trn - DeepJDOT 0, total loss: 94.0929, cat loss: 4.0794, align loss: 90.0135
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:06:06: Val - Source 0, acc: 0.3828, avg acc: 0.1613, cls loss: 1.9711
0it [00:00, ?it/s]

























































































































































305it [05:08,  1.01s/it]
09-10-21 14:11:45: Trn - Source 1, total loss: 1.0120, mixup loss: 1.0120
09-10-21 14:11:45: Trn - Target 1, DefRec loss: 1.4320
09-10-21 14:11:45: Trn - DeepJDOT 1, total loss: 27.0205, cat loss: 3.8644, align loss: 23.1561
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:11:58: Val - Source 1, acc: 0.3929, avg acc: 0.1551, cls loss: 1.9161
09-10-21 14:12:29: Val - Target 1, acc: 0.1890, avg acc: 0.1100, cls loss: 2.3145
























































































































































303it [05:05,  1.02it/s]
09-10-21 14:17:37: Trn - Source 2, total loss: 0.9676, mixup loss: 0.9676
09-10-21 14:17:37: Trn - Target 2, DefRec loss: 1.1267

305it [05:07,  1.01s/it]
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:17:49: Val - Source 2, acc: 0.3981, avg acc: 0.1597, cls loss: 1.7990
09-10-21 14:18:20: Val - Target 2, acc: 0.1899, avg acc: 0.1022, cls loss: 2.2513
























































































































































304it [05:05,  1.00it/s]
09-10-21 14:23:28: Trn - Source 3, total loss: 0.9226, mixup loss: 0.9226
09-10-21 14:23:28: Trn - Target 3, DefRec loss: 0.7980

305it [05:06,  1.01s/it]
09-10-21 14:23:40: Val - Source 3, acc: 0.3728, avg acc: 0.1360, cls loss: 1.7616
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:24:10: Val - Target 3, acc: 0.1931, avg acc: 0.1022, cls loss: 2.1398
























































































































































305it [05:05,  1.00s/it]
09-10-21 14:29:16: Trn - Source 4, total loss: 0.9049, mixup loss: 0.9049
09-10-21 14:29:16: Trn - Target 4, DefRec loss: 0.6871
09-10-21 14:29:16: Trn - DeepJDOT 4, total loss: 12.4601, cat loss: 3.4453, align loss: 9.0149
09-10-21 14:29:29: Val - Source 4, acc: 0.3900, avg acc: 0.1535, cls loss: 1.7002
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:30:01: Val - Target 4, acc: 0.1964, avg acc: 0.1052, cls loss: 2.0984
























































































































































305it [05:06,  1.01s/it]
09-10-21 14:35:09: Trn - Source 5, total loss: 0.8982, mixup loss: 0.8982
09-10-21 14:35:09: Trn - Target 5, DefRec loss: 0.5854
09-10-21 14:35:09: Trn - DeepJDOT 5, total loss: 10.0197, cat loss: 3.3938, align loss: 6.6260
09-10-21 14:35:21: Val - Source 5, acc: 0.3604, avg acc: 0.1237, cls loss: 1.7187
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:35:51: Val - Target 5, acc: 0.2103, avg acc: 0.1064, cls loss: 2.0452

























































































































































305it [05:07,  1.01s/it]
09-10-21 14:40:59: Trn - Source 6, total loss: 0.8790, mixup loss: 0.8790
09-10-21 14:40:59: Trn - Target 6, DefRec loss: 0.4966
09-10-21 14:40:59: Trn - DeepJDOT 6, total loss: 8.1024, cat loss: 3.3062, align loss: 4.7962
09-10-21 14:41:11: Val - Source 6, acc: 0.3523, avg acc: 0.1148, cls loss: 1.6937
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:41:41: Val - Target 6, acc: 0.2209, avg acc: 0.1024, cls loss: 1.9576
























































































































































304it [05:05,  1.03it/s]
09-10-21 14:46:48: Trn - Source 7, total loss: 0.8565, mixup loss: 0.8565
09-10-21 14:46:48: Trn - Target 7, DefRec loss: 0.4478

305it [05:06,  1.00s/it]
09-10-21 14:47:00: Val - Source 7, acc: 0.4669, avg acc: 0.1766, cls loss: 1.5135
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:47:32: Val - Target 7, acc: 0.2979, avg acc: 0.1271, cls loss: 2.0344























































































































































303it [05:04,  1.03it/s]
09-10-21 14:52:39: Trn - Source 8, total loss: 0.8444, mixup loss: 0.8444
09-10-21 14:52:39: Trn - Target 8, DefRec loss: 0.4340

305it [05:06,  1.01s/it]
09-10-21 14:52:52: Val - Source 8, acc: 0.5512, avg acc: 0.2191, cls loss: 1.4152
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:53:24: Val - Target 8, acc: 0.3969, avg acc: 0.1486, cls loss: 1.9837
























































































































































303it [05:05,  1.01s/it]
09-10-21 14:58:32: Trn - Source 9, total loss: 0.8069, mixup loss: 0.8069
09-10-21 14:58:32: Trn - Target 9, DefRec loss: 0.3964

305it [05:07,  1.01s/it]
09-10-21 14:58:44: Val - Source 9, acc: 0.5763, avg acc: 0.1979, cls loss: 1.4746
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 14:59:16: Val - Target 9, acc: 0.4673, avg acc: 0.1552, cls loss: 1.8204

























































































































































303it [05:07,  1.01s/it]
09-10-21 15:04:25: Trn - Source 10, total loss: 0.7861, mixup loss: 0.7861
09-10-21 15:04:25: Trn - Target 10, DefRec loss: 0.3774

305it [05:09,  1.01s/it]
09-10-21 15:04:38: Val - Source 10, acc: 0.6183, avg acc: 0.2419, cls loss: 1.3066
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
Exception ignored in: <function _MultiProcessingDataLoaderIter.__del__ at 0x14b970ac4830>
Traceback (most recent call last):
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1328, in __del__
    self._shutdown_workers()
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1320, in _shutdown_workers
    if w.is_alive():
  File "/home2/vikrant.dewangan/anaconda3/envs/forecasting/lib/python3.7/multiprocessing/process.py", line 151, in is_alive
    assert self._parent_pid == os.getpid(), 'can only test a child process'
AssertionError: can only test a child process
09-10-21 15:05:09: Val - Target 10, acc: 0.4640, avg acc: 0.1638, cls loss: 1.8669





















































































































































